docs/main/docs/executor/become-an-executor/binary-setup/mac/binary-setup.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 1
+---
+
+# Binary Setup
+
+Welcome to the t3rn Executor Setup! This guided process will help you configure your executor with ease, providing step-by-step instructions to ensure a smooth start. Let's get you set up and ready to operate efficiently across multiple blockchain networks.
+
+## Executor Binary - macOS v.0.8.5
+
+### Download Executor Binary
+
+**1.** Download the Executor binary zip file according to your OS from here: https://github.com/t3rn/executor/releases.
+
+**2.** After unzip, in terminal, navigate to the folder that includes the executable file named `executor`.
+
+### Configure Settings and Environment Required Variables
+
+To set the environment variables, copy and paste each command into your terminal. These commands will configure the necessary settings for your Executor to run properly. Make sure you adjust the variable values to your own.
+
+#### GENERAL SETTINGS
+
+**1.** Set your preferred Node Environment. Example:
+
+```bash
+export NODE_ENV=testnet
+```
+
+:::info Devnet Example
+
+```bash
+export NODE_ENV=devnet
+```
+
+:::
+
+**2.** Set your log settings:
+
+```bash
+export LOG_LEVEL=debug
+export LOG_PRETTY=true
+```
+
+#### PRIVATE KEYS
+
+**1.** Set the `PRIVATE_KEY_LOCAL` variable of your Executor, which is the private key of the wallet you will use. Example:
```

Comment:
make note that this key was generated and should not be used, its just an example